doc/stdlib/parallel.rst

```diff
@@ -299,8 +299,6 @@ General Parallel Computing Support
 
    Called by cluster managers using custom transports. It should be called when the custom transport implementation receives the first message from a remote worker. The custom transport must manage a logical connection to the remote worker and provide two ``IO`` objects, one for incoming messages and the other for messages addressed to the remote worker. If ``incoming`` is ``true``\ , the remote peer initiated the connection. Whichever of the pair initiates the connection sends the cluster cookie and its Julia version number to perform the authentication handshake.
 
-.. function:: Future()
```

Comment:
Oh, actually, it's extraneous now that I have the default argument shown in the other `Future` docstring.